src/main/java/janggi/domain/Board.java

```diff
@@ -14,11 +16,24 @@ public final class Board {
 
     private boolean isTeam1Turn = true;
 
-    public Board(final Team team1, final Team team2) {
+    private Board(final Team team1, final Team team2, final boolean isTeam1Turn) {
         validateTeamIsNotNull(team1, team2);
-        validateCountryIsNotSame(team1,team2);
+        validateCountryIsNotSame(team1, team2);
         this.team1 = Team.getFirstTeam(team1, team2);
         this.team2 = Team.getSecondTeam(team1, team2);
+        this.isTeam1Turn = isTeam1Turn;
+    }
+
+    public static Board start(final StartingPosition startingPosition1, final StartingPosition startingPosition2) {
+        final Team team1 = new Team(startingPosition1, new StaticPieceInitializer(), Country.CHO);
+        final Team team2 = new Team(startingPosition2, new StaticPieceInitializer(), Country.HAN);
```

Comment:
새로운 게임을 시작할 때도, 저장된 게임을 불러와서 할 때도 team1 == CHO 인데요!
아래 방식이 더 명확하지 않을까요?

```suggestion
        final Team cho = new Team(startingPosition1, new StaticPieceInitializer(), Country.CHO);
        final Team han = new Team(startingPosition2, new StaticPieceInitializer(), Country.HAN);
```

Reply:
엇 이거는 제 실수 인것 같은데요! team1이 초나라가 아니라 team1이 먼저 시작하는 팀이었어요.
초나라 한나라에 대한 개념은 Team에만 남겨두고 싶어서 그런 것입니다!

새로운 게임을 시작할 때는 초나라가 team1, 불러올 때는 현재 차례가 team1이 되도록 수정했습니다!

[fix: Board의 두 팀이 항상 첫번째로 시작해야 하는 팀부터 설정되도록 리팩터링](https://github.com/woowacourse/java-janggi/pull/86/commits/53c4665bd67058508725d427f9ea641fd92071e9)